src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -38,21 +38,21 @@ export const MyFieldAndBackground = () => {
   const personBackground = person?.background ?? {};
   const personKeywords = person?.keywords ?? [];
 
-  const fieldQuery = useQuery({
+  const keywordsQuery = useQuery({
     enabled: !!debouncedSearchTerm,
-    queryKey: ['field', debouncedSearchTerm],
+    queryKey: ['keywords', debouncedSearchTerm],
     queryFn: () => searchForKeywords(25, 1, debouncedSearchTerm),
-    meta: { message: 'error', variant: 'error' },
+    meta: { message: t('feedback.error.get_keywords'), variant: 'error' },
```

Comment:
vil vel fortsatt krasje her? lurt å teste selv, ved å kastes exception i `searchForKeywords` feks

Reply:
Usikker på hva du mener her. Hva er det som gjær at at den crasher?

Reply:
vi har ikke lagt til noe håndtering av et sånt objekt i `meta`? Ser det ikke krasjer når jeg tester nå, men det har heller ingen effekt.
Prøv med `throw "error"` elns i searchForKeywords for å teste feilhåndtering

Reply:
Sorry, hadde et lite nanodrypp der🤦‍♂️